lib/client/types/errors.go

```diff
@@ -1,4 +1,4 @@
-package client
+package types
```

Comment:
Related to https://github.com/segmentio/aws-okta/pull/243/commits/5e41244927ee326b684b1d30d93adbbd342bfd52?file-filters%5B%5D=.go#r345545238, maybe these errors deserve their own package, but probably not.

Reply:
If the errors types don't have their own package would we create separate errors for MFA and for client. I'm fine with either, was attempting to reduce duplication but I think the current split between client and client/mfa would do the trick.

What do you think?